extension/otto/otto.go

```diff
@@ -354,14 +354,17 @@ func (env *Environment) ClearEnvironment() {
 	env.DataStore.Close()
 }
 
-func throwOtto(call *otto.FunctionCall, exceptionName string, arguments ...interface{}) {
+func ThrowOtto(call *otto.FunctionCall, exceptionName string, arguments ...interface{}) {
 	exception, _ := call.Otto.Call("new "+exceptionName, nil, arguments...)
 	panic(exception)
 }
 
 //ThrowOttoException throws a JavaScript exception that will be passed to Otto
 func ThrowOttoException(call *otto.FunctionCall, format string, arguments ...interface{}) {
-	throwOtto(call, "Error", fmt.Sprintf(format, arguments...))
+	ThrowOtto(call, "Error", fmt.Sprintf(format, arguments...))
+}
+func ThrowTimeOutException(call *otto.FunctionCall, format string, arguments ...interface{}){
+	ThrowOtto(call, "TimeOutException", fmt.Sprintf(format, arguments...))
```

Comment:
TimeOutException should be defined in gohan_core.go (cf. CustomException, ResourceException, etc.)